hw7/hw7_t02.py

```diff
@@ -0,0 +1,36 @@
+"""
+Given two strings.
+
+Return if they are equal when both are typed into
+empty text editors. # means a backspace character.
+
+Note that after backspacing an empty text, the text will continue empty.
+
+Examples:
+    Input: s = "ab#c", t = "ad#c"
+    Output: True
+    # Both s and t become "ac".
+
+    Input: s = "a##c", t = "#a#c"
+    Output: True
+    Explanation: Both s and t become "c".
+
+    Input: a = "a#c", t = "b"
+    Output: False
+    Explanation: s becomes "c" while t becomes "b".
+
+"""
+
+
+def backspace_word_constructor(sequence: str) -> str:
+    skeleton = []
```

Comment:
You use the list in the manner of another data structure and it has the name.  This name will make your intentions more clear if you use it instead of the skeleton. 

Here are some sceletons for you:
https://www.youtube.com/watch?v=-1dSY6ZuXEY

Reply:
Feels like I just got Rickrolled...

Reply:
In a hard(rock) way :)
https://www.youtube.com/watch?v=IdkCEioCp24